### DISCUSSION TITLE
How to change atom's radius in StructureVis module

### DISCUSSION THREAD

**Discussion by thienbinh92** [materialsproject/pymatgen#2749] [Q&A]
Hi, I am using the StructureVis module to visualize my structures of different elements. It shows systems with the same radius for all elements. I would like to ask how to change the atom's radius in the StructureVis module. Is there any way to change the radius's cylinder of bonds as well? Thank you in advance.

**Comment by janosh** [comment]
It would help a lot if you could post a code snippet to show what you currently have so people can jump in and work from there.

**Reply by thienbinh92** [reply]
Hi @janosh, here is the code snippet. I would like to change the radius of the elements and bond cylinders shown. For example, O atoms should have a smaller radius than Ti atoms: r(O) = 0.4, r(Ti)=1.3.
```from pymatgen.vis.structure_vtk import StructureVis
from pymatgen.core.structure import Structure

filename=str("221.vasp")
struct=Structure.from_file(filename)
vis = StructureVis(show_bonds = True,
                   show_polyhedron=False)
vis.set_structure(struct)
vis.show()```

**Reply by janosh** [reply]
Thanks @thienbinh92. Your code can't be run like this. I meant a snippet that includes all the imports and the files necessary. Assuming the actual structure doesn't matter though, you could replace with a dummy structure:

```diff
- filename=str("221.vasp")
- struct=Structure.from_file(filename)
+ from pymatgen.core import Lattice, Structure
+ 
+ structure = Structure(
+     lattice=Lattice.cubic(5),
+     species=("Fe", "O"),
+     coords=((0, 0, 0), (0.5, 0.5, 0.5)),
+ )
```

**Reply by thienbinh92** [reply]
Dear @janosh,
Sorry for that. Maybe this is a testing code:
```from pymatgen.vis.structure_vtk import StructureVis
from pymatgen.core import Lattice, Structure
struct = Structure(lattice=Lattice.cubic(3),species=("Na", "Cl"),coords=((0, 0, 0), (0.5, 0.5, 0.5)))
vis = StructureVis(show_bonds = True,
                   show_polyhedron=False)
vis.set_structure(struct)
vis.show()
```
